auth0_flutter/lib/src/web/auth0_flutter_web_platform_proxy.dart

```diff
@@ -20,8 +20,8 @@ class Auth0FlutterWebClientProxy {
           [final GetTokenSilentlyOptions? options]) =>
       promiseToFuture(client.getTokenSilently(options));
 
-  Future<void> handleRedirectCallback() =>
-      promiseToFuture(client.handleRedirectCallback());
+  Future<RedirectLoginResult> handleRedirectCallback([final String? url]) =>
```

Comment:
The return value and parameters didn't match what is defined in the JS SDK.

For the fix, I need the new return value, but the url is currently unused.
Adding support for custom URL's could be done later, I guess.